src/fileOrder.txt

```diff
@@ -1,5 +1,6 @@
 src/rotate.js
```

Comment:
We could add a helpers.js at the start

which would contain isNumeric and flip.